source/installation/index.html

```diff
@@ -231,12 +231,17 @@ <h2>Compare Installation Methods</h2>
         <td><a href="/common-tasks/os/#backups" target="_blank">Backups</a></td>
         <td>✅</td><td>✅<sup>1</sup></td><td>✅<sup>1</sup></td><td>✅</td>
     </tr>
+    <tr>
+        <td>Managed Restore</td>
+        <td>✅</td><td>❌<sup>2</sup></td><td>❌<sup>2</sup></td><td>✅</td>
+    </tr>
     <tr>
         <td>Managed OS</td>
         <td>✅</td><td>❌</td><td>❌</td><td>❌</td>
     </tr>
 </table>
 
 <sub>1: Backups for Home Assistant Core and Home Assistant Container is provided by the <a href="/integrations/backup" target="_blank"><code>backup</code> integration</a>.</sub>
+<sub>2: Backups for Home Assistant Core and Home Assistant Container is either a tool to migrate to HAOS, or completely manual restore of the backup</sub>
```

Comment:
```suggestion
<sub>2: Backups for Home Assistant Core and Home Assistant Container are either a tool to migrate to HAOS or a completely manual restore of the backup.</sub>
```

Reply:
Thanks for the assistance on proper spelling :)     

I see now that the "footnotes" look funny,,, 
- the link to the backup-integration is not formated as a link anymore - so i guess i broke that one :( 
- and the 2'nd footnote is not starting on a new line (as it should?).
Can you "just fix" that,, or should I have another try with a new PR ?

(newbie on contributing so just figuring out how things work)